controllers/storagecluster/cephrbdmirrors.go

```diff
@@ -57,6 +58,16 @@ func (r *StorageClusterReconciler) newCephRbdMirrorInstances(initData *ocsv1.Sto
 			Spec: cephv1.RBDMirroringSpec{
 				Count:     1,
 				Resources: defaults.GetDaemonResources("rbd-mirror", initData.Spec.Resources),
+				Placement: cephv1.Placement{
```

Comment:
We allow user defined placements for all daemons. If placement for rbd-mirror is not supported in Rook, it should be added there first. Then it should be added to ocs-operator in this file (https://github.com/red-hat-storage/ocs-operator/blob/main/controllers/defaults/placements.go). Then in this line, we'll need to merge user defined placement with our defaults.